### ISSUE TITLE
Plugin not compatible with cakephp 3.6.X

### ISSUE THREAD

**Issue by avanimanvar** [multidots/cakephp-rest-api#30]

**Comment by zhunt** [comment]
I got it working in 3.6 by:

In **src/Application.php** comment out the line: $this->addPlugin('RestApi', ['bootstrap' => true]);

then add it to **/config/bootstrap.php** like: Plugin::load('RestApi', ['bootstrap' => true]);

**Comment by narendravaghela** [comment]
@avanimanvar what issues are you facing with 3.6.x?

**Comment by avanimanvar** [comment]
Some functions are deprecated and show the warnings due to that.

**Comment by narendravaghela** [comment]
Hmm, could you make a PR with fixes @avanimanvar? And ask @dkvasani to merge it :)

**Comment by avanimanvar** [comment]
I have already done it. But merging is pending by @dkvasani :). He will do the QA first and then merge it.

**Comment by richcode** [comment]
I tried I got problem with CakePHP 3.6.7
Its working in CakePHP 3.5.6

The error is
Failed to load https://****/ login: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://***.com' is therefore not allowed access.

although my api.php is
'origin' => '*',

when I switch to CakePHP version it happens

**Comment by dkvasani** [comment]
Not this is CakePHP 3.6 compitable.
